backend/Bindings/Edna.Bindings.LtiAdvantage/Services/AccessTokenService.cs

```diff
@@ -9,6 +9,7 @@
 using System.Net.Http;
 using System.Security.Claims;
 using System.Threading.Tasks;
+using Edna.Bindings.LtiAdvantage.Utils;

```

Comment:
Same question as above.

Reply:
Essentially, in order to use the custom handler implementing the server certificate validation logic, we need to pass it a string indicating the `Name` of the handler. All these imports have been made in order to use the `Name`  property of the new Handler (or initialize one in cases where we are using HttpClient directly).